docs/README.md

```diff
@@ -119,50 +117,13 @@ Now open your browser to [localhost:8000](http://localhost:8000) and select the
 
 If you go back to the browser and start a chat with Qwen2, you will notice that it will take a while to respond at first. This is because we set `minReplicas: 0` for this model and KubeAI needs to spin up a new Pod (you can verify with `kubectl get models -oyaml qwen2-500m-cpu`).
 
-## Documentation
```

Comment:
I think we should highlight our full documentation before the Local Quickstart guide.